setup.py

```diff
@@ -25,8 +25,8 @@
     classifiers=[
```

Comment:
this came from @adamchainz 's package. I can probably remove it but maybe he can shed some light?